homeassistant/components/zwave/node_entity.py

```diff
@@ -116,8 +122,27 @@ def network_node_changed(self, node=None, value=None, args=None):
                 value.command_class == COMMAND_CLASS_CENTRAL_SCENE):
             self.central_scene_activated(value.index, value.data)
 
+        if value is not None:
+            self.fire_value_changed_event(value)
+
+
         self.node_changed()
 
+    def fire_value_changed_event(self, value):
+        """Fire a value_changed event when a value changes."""
+        self.hass.bus.fire(EVENT_VALUE_CHANGED_EVENT, {
+            ATTR_ENTITY_ID:           self.entity_id,
+            ATTR_NODE_ID:             self.node_id,
+            ATTR_VALUE_INDEX:         value.index,
+            ATTR_VALUE_DATA:          value.data,
+            ATTR_VALUE_INSTANCE:      value.instance,
+            ATTR_VALUE_COMMAND_CLASS: value.command_class,
+            ATTR_VALUE_GENRE:         value.genre,
+            ATTR_VALUE_IS_SET:        value.is_set,
+            ATTR_VALUE_LABEL:         value.label,
```

Comment:
We shouldn't report static properties like label and is_polled. Now that I'm looking at this, I'd propose we slim it down to only entity_id, node_id, value_id, and value_data.